Resources/views/CRUD/base_history.html.twig

```diff
@@ -39,7 +39,7 @@ file that was distributed with this source code.
                             <tr class="{% if (currentRevision != false and revision.rev == currentRevision.rev) %}current-revision{% endif %}">
                                 <td>{{ revision.rev}}</td>
                                 <td>{% include admin.getTemplate('history_revision_timestamp') %}</td>
-                                <td>{{ revision.username}}</td>
+                                <td>{{ revision.username|default('label_anonymus_user'|trans({}, 'SonataAdminBundle')) }}</td>
```

Comment:
Maybe could use ``System`` ?

otherwise we can make an empty translation target, but then you are able to define it